<launch>
	<node 
		pkg = "temperature_sensor"
		type = "start.py"
		name = "temperature_sensor"
		output = "screen"
	>
		<param name="port" value="/dev/ttyUSB1" />
		<param name="slave_adress" value="16" />
		<param name="baudrate" value="9600" />
		<param name="capture_time" value="10" />
		<param name="publication_period" value="1" />
	</node>
</launch>